lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -154,10 +156,10 @@ def post(self):
             # fetch the user data
             user = UserModel.query.filter_by(username=post_data.get("username")).first()
 
-            if user and self.bcrypt.check_password_hash(
+            if user and self.app.bcrypt.check_password_hash(
```

Comment:
It'd be cleaner to encapsulate `bcrypt` inside the `UserModel` class and export a (class?) method to do this. That is, from the outside of the `UserModel`, this is just a method on the user object/class.